vendor/bat-native-ads/src/bat/ads/internal/account/account_util_unittest.cc

```diff
@@ -64,7 +64,7 @@ TEST_F(BatAdsAccountUtilTest, ResetRewards) {
   privacy::GetUnblindedPaymentTokens()->AddTokens(unblinded_payment_tokens);
 
   // Act
-  ResetRewards([](const bool success) {
+  ResetRewards(base::BindOnce([](const bool success) {
```

Comment:
maybe it's a task for another PR, but we should replace such callbacks with `MockCallback`. Right now it's impossible to detect if the callback was actually called or not. IMO, this is critical, especially in such PRs when we refactor the callback foundation.

it should become something like this:
```
base::MockCallback<ResetRewardsCallback> callback;
EXPECT_CALL(callback, Run(true)).WillOnce([](const bool success) {
   ...
});
ResetRewards(callback.Get());
```

Reply:
filed the issue: https://github.com/brave/brave-browser/issues/28056